### PR TITLE
Add docs for creating new avatars

### LINKED ISSUE #19: Add option to add new avatars
It would be useful to add an option under the avatar tab to add new avatars.

### PR DESCRIPTION
**Pull request by dmey**
Closes #19.

### CODE REVIEW

File: AVATARS.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+<!-- omit in toc -->
+## Table of contents
+
+- [Prerequisites](#prerequisites)
+- [Generate from scratch (work in progress)](#generate-from-scratch-work-in-progress)
+- [Generate from an existing design](#generate-from-an-existing-design)
+
+## Prerequisites
+
+- [Inkscape => 1.0](https://inkscape.org/release/)
```
Comment (letmaik) [inline AVATARS.md]:
```suggestion
- [Inkscape >= 1.0](https://inkscape.org/release/)
```

File: AVATARS.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+```
+    [Layer 1]
+    |---- skeleton
+    |---- illustration
+            |---- path 1
+            |---- path 2
+            |---- path 3
+```
+3. Embed the sample skeleton in `skeleton` group into your illustration by moving the joints around.
+4. Save as SVG with name.
```
Comment (letmaik) [inline AVATARS.md]: incomplete sentence?

File: AVATARS.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+5. TODO: Open Avatar Animator and load the new SVG from the `avatar` tab.
+
+## Generate from an existing design
+
+1. Download and open an existing illustration from the [illustration folder](resources/illustration) in Inkscape and create a copy, then close sample skeleton SVG. **Note**: do not add, remove or rename joints (circles) in the group. Avatar Animator relies on these named paths to read the skeleton’s initial position. Missing joints will cause errors.
+2. From `Object` > `Objects...` menu, expand the `illustration` layer. From here you can replace path elements with your own ones. Make sure to flatten all subgroups so that the layer `illustration` only contains path elements, composite paths are not supported at the moment. Notice that some path elements are grouped together, for example eyes, jaw, nose. Make sure to keep these groups and only replace the path elements inside these groups.
+3. Save as SVG with name.
```
Comment (letmaik) [inline AVATARS.md]: incomplete sentence?

File: AVATARS.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    |---- illustration
+            |---- path 1
+            |---- path 2
+            |---- path 3
+```
+3. Embed the sample skeleton in `skeleton` group into your illustration by moving the joints around.
+4. Save as SVG with name.
+5. TODO: Open Avatar Animator and load the new SVG from the `avatar` tab.
+
+## Generate from an existing design
```
Comment (letmaik) [inline AVATARS.md]: Does this scenario even make sense? Is it easier to replace all existing paths instead of starting from scratch?

Comment (dmey) [reply AVATARS.md]: I am really not an expert with this and I am sure there is an easier way, but from what I have tried so far creating the groups was not straightforward and there may be something missing from the text above -- hence the "work in progress" in the heading... The one that I know works is from existing design.

File: AVATARS.md (hunk 2 above)
Comment (dmey) [reply AVATARS.md]: Changed in https://github.com/letmaik/avatar-animator/pull/20/commits/a170ebe45dc4f7aa0c705d694e1ba627b41be7f5

File: AVATARS.md (hunk 3 above)
Comment (dmey) [reply AVATARS.md]: Changed in https://github.com/letmaik/avatar-animator/pull/20/commits/a170ebe45dc4f7aa0c705d694e1ba627b41be7f5

File: AVATARS.md (hunk 4 above)
Comment (letmaik) [reply AVATARS.md]: Sure, but I think the scenario is not to "generate" something new but rather tweak an existing design, right? Wasn't that what you were doing?

Comment (dmey) [reply AVATARS.md]: I have changed this in https://github.com/letmaik/avatar-animator/pull/20/commits/bd33a467c2e00734c1177186598136db5a615ce7 -- is this clearer?

Comment (letmaik) [reply AVATARS.md]: Yes, that's better.

**Comment by letmaik** [comment]
Can you mark this as draft PR if it's not ready to merge?

**Comment by dmey** [comment]
> Can you mark this as draft PR if it's not ready to merge?

I don't think you can change it once it's been created

**Comment by letmaik** [comment]
On the top right, I changed it for you.

**Comment by dmey** [comment]
> On the top right, I changed it for you.

I cannot see it -- I think I don't have the permissions for it. Can you be a bit more specific?

**Comment by letmaik** [comment]
Once it's a draft PR there's nothing at the top right, but instead just the big button "Ready for review" below the comments. Feel free to switch this PR around to see what happens.

File: README.md [hunk 5]
```diff
@@ -48,6 +49,11 @@ From the Avatar Animator folder, launch `avatar-animator.exe` before your prefer
 | Slack           | Start a call, then click on the gear icon in the top left corner of the call window and select `OBS-Camera` from the `Camera` drop-down menu. | TODO       |
 
 
+## How to create or add new avatars
+
+To create new avatars, see [AVATARS.md](AVATARS.md). If have already created one before and would like to add it Avatar Animator, simply drag and drop the SVG into the Avatar Animator app.
```
Comment (letmaik) [inline README.md]:
```suggestion
To create new avatars, see [AVATARS.md](AVATARS.md). If you have already created one before and would like to add it to Avatar Animator, simply drag and drop the SVG into the Avatar Animator app.
```

**Review by letmaik — APPROVED** [review]

**Pull request merged**